tools/samtools_fastq.cwl

```diff
@@ -19,29 +19,36 @@ arguments:
       set -eo pipefail
```

Comment:
So this pipefail isn't going to work.

Reply:
Does it ever? I think I need a refresher/link to your previous work. Should we turn the command into a "shell script" with that in the header?